xds/src/main/java/io/grpc/xds/Filter.java

```diff
@@ -104,7 +104,7 @@ private ConfigOrError(String errorDetail) {
   }
 
   /** Filter config with instance name. */
-  final class NamedFilterConfig {
+  public final class NamedFilterConfig {
```

Comment:
"Modifier 'public' is redundant for inner classes of interfaces "  (from the IDE)

Reply:
It doesn't hurt to have it, though.